src/Faithlife.Build/DotNetBuild.cs

```diff
@@ -221,394 +221,407 @@ IReadOnlyList<string> BuildNuGetPackages()
 			return createdPackagePaths;
 		}
 
+		build.Target("publish-nuget-output")
+			.Describe("Publishes NuGet packages and documentation from --nuget-output. Useful for skipping all builds.")
```

Comment:
`Publishes all NuGet packages at --nuget-output.` might be clearer. (This is a dangerous command for users that collect lots of NuGet packages in an output directory.)

I don't know what to say about documentation. `DoPublish` certainly does not "publish documentation from --nuget-output". Publishing documentation requires a build.

Reply:
That's a great point...

I think there is value in supporting a CI workflow that can publish packages without pushing docs at the same time, but could also contain an additional step to build and push docs.

For example, a distributed workflow might split multiple build/package steps and even more platform-test steps, and after all the steps are successful, resume to push packages and even rebuild (or, restore an archived patch) to push the docs. In this case, it might be important to publish the same artifacts that tests were run against, versus rebuilding the project. Further, archiving and restoring a patch could avoid a successive build, too, though it might require another change to support that workflow within FL.Build (if that is important).

However, I could also be convinced that the due to this complexity, it might be preferable to avoid FL.Build to push NuGet packages and tag the repository. (The tag logic is the main reason to use FL.Build for this step).